### PR TITLE
Highlight default tier and show inline compare button on selected tier

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Tier selector improvements
- Add "Popular" label to default tier (now hardcoded as STANDARD)
- Add inline button "See what's included" to selected tier

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4488?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4488** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4488?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4487** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4487?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4484** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4484?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4482** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4482?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4481** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4481?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4479** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4479?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4478** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4478?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/CardRadioGroup.tsx [hunk 1]
```diff
@@ -14,8 +14,8 @@ export function Root({ children, className, ...forwardedProps }: RadioGroupProps
 
 export function Item({ children, className, value, ...forwardedProps }: RadioGroupItemProps) {
   return (
-    <RadioGroup.Item value={value} className={clsx(item, className)} {...forwardedProps}>
-      {children}
+    <RadioGroup.Item value={value} {...forwardedProps} asChild>
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/CardRadioGroup.tsx:17]: It used to render button by default which is not compatible with nested buttons

File: apps/store/src/features/priceCalculator/CardRadioGroup.css.ts [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
+    selectors: {
+      '&[data-state=checked]': {
+        backgroundColor: tokens.colors.buttonPrimary,
+        vars: {
+          [tokens.colors.textPrimary]: tokens.colors.textNegative,
+          [tokens.colors.textSecondary]: tokens.colors.gray500,
+        },
       },
     },
+    vars: {
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/CardRadioGroup.css.ts:21]: Some custom overrides for now, but we should have a button variant that looks right on dark background

**Review by gustaveen — APPROVED** [review]
GJ!

**Pull request merged**